lib/studies/reply/mail_view_page.dart

```diff
@@ -14,26 +15,41 @@ class MailViewPage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final galleryTheme = GalleryOptions.of(context).themeMode;
+    bool isDark;
+
+    if ((galleryTheme == ThemeMode.dark) | (galleryTheme == ThemeMode.light)) {
+      isDark = GalleryOptions.of(context).themeMode == ThemeMode.dark;
+    } else {
+      isDark = MediaQuery.of(context).platformBrightness == Brightness.dark;
+    }
+
     return Scaffold(
       body: SafeArea(
-        child: Padding(
-          padding:
-              const EdgeInsetsDirectional.only(top: 42, start: 20, end: 20),
-          child: Column(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              _MailViewHeader(
-                subject: email.subject,
-                sender: email.sender,
-                recipients: email.recipients,
-                time: email.time,
-                avatar: email.avatar,
-              ),
-              const SizedBox(height: 32),
-              Expanded(
-                child: _MailViewBody(message: email.message),
-              ),
-            ],
+        bottom: false,
+        child: Container(
+          width: double.infinity,
+          height: double.infinity,
```

Comment:
I think this might work without specifying width and height since you have an expanded widget